common/nymsphinx/src/receiver.rs

```diff
@@ -7,6 +7,8 @@ use crypto::asymmetric::encryption;
 use crypto::shared_key::recompute_shared_key;
 use crypto::symmetric::stream_cipher;
 use crypto::symmetric::stream_cipher::CipherKey;
+use nym_outfox::error::OutfoxError;
+use nym_outfox::lion::lion_transform_decrypt;
```

Comment:
I'd argue `nym_outfox` stuff shouldn't be inside `nymsphinx` crates. Instead there should be some common crate that those two depend on it. Idk, maybe `common/packets` or something?

Reply:
Yeah, we'll evolve to that for sure